contrib/ide/vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "rust-analyzer.cargo.allFeatures": true
-}
+    "rust-analyzer.cargo.allFeatures": true,
+    "rust-analyzer.checkOnSave.command": "ruma-lint"
+}
```

Comment:
Missing final newline. You should set up an [editorconfig](https://editorconfig.org/) extension for your editor / IDE ;)